usps/main.py

```diff
@@ -0,0 +1,112 @@
+"""
+USPS Microservice FastAPI Web App.
+"""
+
+import json
+import logging
+import ssl
+import base64
+from typing import Optional
+from uuid import UUID
+from http import HTTPStatus
+from fastapi import FastAPI, Header
+from fastapi.responses import JSONResponse
+from aiohttp import ClientSession, ClientError
+from pydantic import BaseModel
+from google.oauth2 import service_account
+from google.auth.transport.requests import Request
+from usps.config import settings
+from usps import transaction_log
+
+app = FastAPI()
+
+
+class AddressVerificationInfo(BaseModel):
+    """
+    Request model used for generating documentation and input validation.
```

Comment:
We're using FastAPI and taking advantage of Pydantic for input validation/documentation generation. Developers should be familiar with these tools (why we're using them and what they do) before touching the code. We should therefore reserve comments/docstrings for explaining/documenting **our application**, rather than the tools we use (which would be better off as a section in the readme.

So to me, this docstring would be better as something like: `Request model for the USPS AII /confidence_indicator API`